app/controllers/quotes_controller.rb

```diff
@@ -1,6 +1,13 @@
 class QuotesController < ApplicationController
   def new
     @quote = Quote.new
+    @lead = Lead.new
+    @service_expedition_time = [
+      {label: 'URGENT in ', id: '3hrs', price: 99.99},
+      {label: 'In the next', id: '8hrs', price: 49.99},
+      {label: 'FREE', id: '24hrs', price: 0},
+    ]
+    @salt_bag_price = 35
```

Comment:
FYI all amounts should be coming from the controller, not just the salt bags (min charge, price per sqft, etc.)

Reply:
done